### PR TITLE
Watch file changes

### PR DESCRIPTION
**Pull request by gingerich**
- Volume mount to watch host file changes inside container
- Improved package scripts
- Track package-lock.json

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,12 +12,7 @@ TWILIO_AUTH_TOKEN=
 ### Run Locally
 To run the API locally you need first to build and run docker container:
 
-    docker-compose build
-    docker-compose up
-
-Then from a seperate terminal window you need to run `npm run dev` to watch the changes.
```
Comment (gingerich) [inline README.md]: No longer needed since file changes are watched from within container

File: docker-compose.yml [hunk 2]
```diff
@@ -10,21 +10,16 @@ services:
...
       - MYSQL_HOSTNAME=db
-      - MYSQL_PORT=3306
-      - MYSQL_DB=sanctuary_dev
     ports:
       - "80:8080"
-    # volumes:
-    #   - .:/home/node/app
-    #   - node_modules:/home/node/app/node_modules
+    volumes:
+      - .:/app
```
Comment (gingerich) [inline docker-compose.yml]: mount working directory into container

File: package.json [hunk 3]
```diff
@@ -5,11 +5,12 @@
   "main": "dist",
   "scripts": {
     "dev": "nodemon -w src --exec \"babel-node src\"",
-    "migrate": "knex migrate:latest --knexfile knexfile.js",
+    "migrate": "node -r dotenv/config ./node_modules/knex/bin/cli.js migrate:latest",
```
Comment (gingerich) [inline package.json]: load .env values before running migration (knexfile requires values from `process.env`)

Comment (iamstevedavis) [reply package.json]: :+1:

**Pull request merged**